resources/js/components/open/forms/components/form-components/FormCustomCode.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-  <collapse class="p-5 w-full border-b" :default-value="false">
+  <collapse class="py-3 w-full border-b" :default-value="false">
     <template #title>
       <h3 class="font-semibold text-lg">
-        <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5 inline text-gray-500 mr-2 -mt-1" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+        <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5 inline text-nt-blue mr-2 -mt-1" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Comment:
`text-nt-blue` should not exist anymore

Reply:
![image](https://user-images.githubusercontent.com/103994754/197115090-36529602-775b-464b-8b35-1a82e3b2115b.png)

Class is valid and working also